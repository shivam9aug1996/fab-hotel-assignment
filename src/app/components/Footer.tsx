import React from "react";

const Footer = () => {
  return (
    <footer className="bg-gray-800 text-white py-4">
      <div className="max-w-4xl mx-auto text-center">
        <p className="text-sm">
          &copy; {new Date().getFullYear()} Fab Hotels. All rights reserved.
        </p>
      </div>
    </footer>
  );
};

export default Footer;
