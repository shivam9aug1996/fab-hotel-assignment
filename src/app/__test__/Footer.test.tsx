import Footer from "@/app/components/Footer";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("Footer", () => {
  it("renders footer", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
