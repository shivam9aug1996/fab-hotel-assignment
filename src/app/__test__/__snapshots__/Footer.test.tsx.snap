// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders footer 1`] = `
<div>
  <footer
    class="bg-gray-800 text-white py-4"
  >
    <div
      class="max-w-4xl mx-auto text-center"
    >
      <p
        class="text-sm"
      >
        Â© 
        2024
         Fab Hotels. All rights reserved.
      </p>
    </div>
  </footer>
</div>
`;
